gradle-plugins/rnbrownfield-gradle-plugin/rnbrownfield-plugin/src/main/kotlin/com/callstack/rnbrownfield/plugin/ManifestMerger.kt

```diff
@@ -0,0 +1,85 @@
+package com.callstack.rnbrownfield.plugin
```

Comment:
Could we use slightlty different package name:
```suggestion
package com.callstack.react.brownfield.plugin
```

(this is a general comment for all packages)